### PR TITLE
Fix angular directive bugs

### LINKED ISSUE #1254: Investigate angular app memory leak
https://github.com/tastejs/todomvc/pull/1223#discussion_r27026063

### PR DESCRIPTION
**Pull request by samccone**
Fixes #1254

cleaned up a few minor things as I went along

### CODE REVIEW

**Comment by samccone** [comment]
@arthurvr any thoughts on this one?

File: examples/typescript-angular/js/Application.js [hunk 1]
```diff
@@ -44,9 +44,8 @@ var todos;
     function todoBlur() {
         return {
             link: function ($scope, element, attributes) {
-                element.bind('blur', function () {
-                    $scope.$apply(attributes.todoBlur);
-                });
+                element.bind('blur', function () { $scope.$apply(attributes.todoBlur); });
+                $scope.$on('$destory', function () { element.unbind('blur'); });
```
Comment (arthurvr) [inline examples/typescript-angular/js/Application.js]: destory -> destroy

File: examples/typescript-angular/js/directives/TodoBlur.ts [hunk 2]
```diff
@@ -10,6 +10,7 @@ module todos {
         return {
             link: ($scope: ng.IScope, element: JQuery, attributes: any) => {
                 element.bind('blur', () => { $scope.$apply(attributes.todoBlur); });
+                $scope.$on('$destory', () => { element.unbind('blur'); });
```
Comment (arthurvr) [inline examples/typescript-angular/js/directives/TodoBlur.ts]: typo: destory -> destroy

File: examples/typescript-angular/js/Application.js (hunk 1 above)
Comment (samccone) [inline examples/typescript-angular/js/Application.js]: Whoooops

**Comment by samccone** [comment]
kk @arthurvr thanks and updated

**Comment by arthurvr** [comment]
In typescript-angular this behaviour with the escape key appears to be broken (in chrome). Already broken on master it seems, can somebody confirm that?

**Comment by samccone** [comment]
I can confirm :white_check_mark: it is broken

**Comment by arthurvr** [comment]
I'm opening an issue about it.

**Comment by arthurvr** [comment]
=> https://github.com/tastejs/todomvc/issues/1264

Let's land this one, then. :rocket:

**Comment by samccone** [comment]
:zap: :zap: :zap: :zap: :zap:

**Pull request merged**